src/main/java/productDependency/CreateCustomerPage.java

```diff
@@ -0,0 +1,172 @@
+import webDataPages.SignupChildCompanyInvoiceAsSellerPage;
+import org.openqa.selenium.support.ui.Select;
+
+public class CreateCustomerPage extends BasePage {
+	
+	public CreateCustomerPage(WebDriver webdriver) {
+		super(webdriver);
+	}
+
+	Logger log = Logger.getLogger(CreateCustomerPage.class);
+
+	PropertyValExtractors p = new PropertyValExtractors();
+	Actions actions = new Actions(driver);
+	String sheetName = "AddCustomer";
+	String xlsxName = "/ProductDependency_TestData.xlsx";
+
+	@FindBy(how = How.XPATH, using = "//input[@name='j_username']")
+	private WebElement enterLoginID;
+
+	/**
+	 * Method to enter login ID.
+	 * 
+	 * @throws Exception
+	 */
+	public void enterLoginID() throws Exception {
+		CreateCustomerPage sp = new CreateCustomerPage(driver);
+		log.info("Verifying the Login ID is available or not");
+		Assert.assertTrue(enterLoginID.isDisplayed());
+		enterLoginID.sendKeys(BasePage.getCellData(xlsxName, sheetName, 0, 0));
+	}
+
+	@FindBy(how = How.XPATH, using = "//input[@name='j_password']")
+	private WebElement enterPassword;
+
+	/**
+	 * Method to enter Password.
+	 * 
+	 * @throws Exception
+	 */
+	public void enterPassword() throws Exception {
+		CreateCustomerPage sp = new CreateCustomerPage(driver);
+		log.info("Verifying the First Name is available or not");
+		Assert.assertTrue(enterPassword.isDisplayed());
+		enterPassword.sendKeys(BasePage.getCellData(xlsxName, sheetName, 1, 0));
+
+	}
+
+	private WebElement selectCompany;
+
+	/**
+	 * Method to select Company.
+	 * 
+	 * @throws IOException
+	 */
+	public void selectCompany() throws Exception {
+		CreateCustomerPage sp = new CreateCustomerPage(driver);
+		WebElement Companyelement = driver.findElement(By.xpath("//select[@name='j_client_id']"));
+		Select se = new Select(Companyelement);
+		se.selectByVisibleText(BasePage.getCellData(xlsxName, sheetName, 2, 0));
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit save']")
+	private WebElement clickLoginButton;
+
+	/**
+	 * Method to Click on Save Changes Button
+	 */
+	public void clickLoginButton() {
+		CreateCustomerPage sp = new CreateCustomerPage(driver);
+		log.info("Verifying the login button is available or not");
+		Assert.assertTrue(clickLoginButton.isDisplayed());
+		clickLoginButton.click();
+	}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='menu.link.customers']/a")
+	private WebElement clickCustomerTab;
+
+	/**
+	 * Method to click on Products tab after successful login.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickCustomerTab() throws IOException {
+		CreateCustomerPage sp = new CreateCustomerPage(driver);
+		log.info("Click on Products Tab after successful login");
+		Assert.assertTrue(clickCustomerTab.isDisplayed());
+		clickCustomerTab.click();
+		JavaScriptExec.sleep();
+}
+	public void createCustomer() throws IOException {
+		CreateCustomerPage sp = new CreateCustomerPage(driver);
+		for (int i = 0; i < 8; i++) {
+			WebElement clickAddNewButton = driver
```

Comment:
@mayankchhajer please remove the extra cycle from the loop, you have to create only 7 customer and you added the 8 cycles in the loop.

Reply:
Done